Lib/re.py

```diff
@@ -154,31 +155,6 @@ class RegexFlag(enum.IntFlag):
     TEMPLATE = T = sre_compile.SRE_FLAG_TEMPLATE # disable backtracking
     DEBUG = sre_compile.SRE_FLAG_DEBUG # dump pattern after compilation
 
-    def __repr__(self):
-        if self._name_ is not None:
-            return f're.{self._name_}'
-        value = self._value_
-        members = []
-        negative = value < 0
-        if negative:
-            value = ~value
-        for m in self.__class__:
-            if value & m._value_:
-                value &= ~m._value_
-                members.append(f're.{m._name_}')
-        if value:
-            members.append(hex(value))
-        res = '|'.join(members)
-        if negative:
-            if len(members) > 1:
-                res = f'~({res})'
-            else:
-                res = f'~{res}'
-        return res
-    __str__ = object.__str__
-
-globals().update(RegexFlag.__members__)
-
```

Comment:
Nice win to delete/move all this code!

Reply:
Thanks.  It took me a few iterations to figure out how I wanted it, and some insights from Serhiy helped a great deal.